### ISSUE TITLE
Customizable ordering of components on source page

### ISSUE THREAD

**Issue by jadalilleboe** [skyportal/skyportal#3573]
Potentially make the order of the components on the source page configureable in the config
@guynir42 suggested changing the ordering without asking and seeing who complains (i.e. #3565)

**Comment by jadalilleboe** [comment]
I agree, I think @profjsb also suggested yesterday A/B testing for the positioning of the plots and thumbnails to see what makes the most sense. but definitely a "low priority" issue

**Comment by guynir42** [comment]
@stefanv We've discussed this last meeting and I believe this shouldn't require much complexity. Might be that we can get away with just replacing the order of components but probably some people will be upset that we moved their proverbial cheese. If it is configurable then everyone is happy. 

But I agree if this turns out to be a big code investment we should reconsider (and instead just change the order and see who complains).

**Comment by stefanv** [comment]
@guynir42 It's a fine balance, catering to users vs ensuring that you don't eventually get overwhelmed by maintenance burden resulting from lots of subtle features. Death by a thousand paper cuts and all that. Either way, you know better than I whether the trade-off is worth it.
